releases/goerli/1.9.5/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+	<head>
+		<title>Goerli Release 1.9.5</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Goerli Release 1.9.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/goerli/1.9.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmcHJSEKK5jEoVMNFqfujht33ZL75r5bEbpsp6URBhmRLT">Contract addresses (IPFS hosted)</a></li>
```

Comment:
this doesn't match the contracts in the PR.

Reply:
Good catch I had forgotten to update the RWAs PIPs. It's replaced now.